### DISCUSSION TITLE
New Theme Importer

### DISCUSSION THREAD

**Discussion by fuzenco** [zed-industries/zed#7340] [Feature Requests]
I’ve not tried this but I just ran across this theme importer:

https://twitter.com/_yogr/status/1753197596808511593

**Comment by michealp-coder** [comment]
Any plans to make this functionality exist natively within Zed?

That way you don't have to have Xcode, Postgres, etc installed just to build Zed to do this import?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
